source/renderer/app/components/staking/stake-pools/StakePoolsList.js

```diff
@@ -4,23 +4,32 @@ import { observer } from 'mobx-react';
 import React, { useEffect, useState } from 'react';
 import type { ElementRef } from 'react';
 import { AutoSizer, List, WindowScroller } from 'react-virtualized';
-import { hideAll } from 'tippy.js';
+import { Instance } from 'tippy.js';
 import LoadingSpinner from '../../widgets/LoadingSpinner';
 import styles from './StakePoolsList.scss';
 import StakePool from '../../../domains/StakePool';
 import { ThumbPool } from '../widgets/ThumbPool';
 
+type TippyElement = Element & {
+  _tippy: Instance,
+};
+
 // Maximum number of stake pools for which we do not need to use the preloading
 const PRELOADER_THRESHOLD = 100;
 const POOL_THUMB_SIZE = 80;
 const POOL_THUMB_GRID_GAP = 10;
 
 /**
- * Utility function to programmatically hide all pop overs
+ * Utility function to programmatically hide the active pop over
  * This is used to hide the pool tooltips on scrolling the list
  */
-function hideAllPopOvers() {
-  hideAll();
+function hidePoolPopOver() {
```

Comment:
This was also closing the discreet mode topbar pop over. We needed a way to only target pop overs in the staking pools screen. There is a possible better approach by using tippy.js singleton mode, but I was not able to make it work with the current popover from react-polymorth. It would be good to make changes to react-polymorth and make the singleton option possible to use.

Reply:
careful with that regarding performance issues

Reply:
Do you mean from using tippy singleton mode or for the current change to hide stake pool pop over?

Reply:
regarding tippy singleton mode – i think i tried many things including singleton mode and there have been some issues (but i don't remember exactly what it was) … of course you can also try/test it, just wanted to remind to be careful regarding performance because i remember that there was an issue/limitations with singletons.

Reply:
Ok, I'll keep that in mind. For now I decided to not follow on singleton approach since it would require direct use on the react-tippy package. I suggested for us to increment the current react-polymorth pop over api to make it possible to use singleton mode in general, but in the scope of another task.